lib/rubocop/cop/lint/ambiguous_regexp_literal.rb

```diff
@@ -21,21 +21,20 @@ module Lint
       #
       #   # With parentheses, there's no ambiguity.
       #   do_something(/pattern/i)
-      class AmbiguousRegexpLiteral < Cop
+      class AmbiguousRegexpLiteral < Base
         include ParserDiagnostic
```

Comment:
My 2 cents is that this module doesn't do enough to justify its existence and makes things more complicated than if it didn't exist.